src/common/components/core/MultipleSelect.tsx

```diff
@@ -7,6 +7,7 @@ import IconButton from '@mui/material/IconButton';
 import Box from '@mui/material/Box';
 
 import WrapperTypography from './WrapperTypography';
+import { trim, uniqBy } from 'lodash';
```

Comment:
Use javascript native .trim() instead of lodash.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim